vm/src/stdlib/ast/node.rs

```diff
@@ -44,7 +45,7 @@ impl<T: Node> Node for Box<T> {
         source_code: &SourceCodeOwned,
         object: PyObjectRef,
     ) -> PyResult<Self> {
-        T::ast_from_object(vm, source_code, object).map(Box::new)
+        T::ast_from_object(vm, source_code, object).map(Self::new)
```

Comment:
This is logically right but lookins so unfamiliar 😲

Reply:
It can be reverted, just say the word:)